namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-dev/resources/route53.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_route53_zone" "example_team_route53_zone" {
```

Comment:
Could you rename your resource please. `example_team` is not a suitable name

Reply:
and all references below

Reply:
As soon as I change it it goes red in my IDE...

Reply:
Suggest the web page puts that in square brackets to denote a required change